crates/language_model/src/request.rs

```diff
@@ -294,7 +294,10 @@ impl LanguageModelRequest {
                 top_p: None,
                 top_k: None,
             }),
-            safety_settings: None,
+            safety_settings: Some(vec![google_ai::SafetySetting {
+                category: google_ai::HarmCategory::DangerousContent,
+                threshold: google_ai::HarmBlockThreshold::BlockNone,
```

Comment:
Not blocking _any_ dangerous content seems like the wrong choice for a default.

Reply:
Should there be an option for adjusting the filter settings?

`HarmCategory::DangerousContent` is one of the 5 categories. The filter currently blocks content in all other categories. 

From Gemini API Docs:
<details> 
  <summary>The Gemini API's adjustable safety filters cover the following categories:</summary>

Category | Description
-- | --
Harassment | Negative or harmful comments targeting identity and/or protected attributes.
Hate speech | Content that is rude, disrespectful, or profane.
Sexually explicit | Contains references to sexual acts or other lewd content.
Dangerous | Promotes, facilitates, or encourages harmful acts.
Civic integrity | Election-related queries.

</details>